exercises/hamming/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 /// Return the Hamming distance between the strings,
 /// or an error if the lengths are mismatched.
```

Comment:
is "an error" still OK here? It's not clear that `None` is "an error", but since the word "error" is not capitalised (i.e. it is not explicitly referring to `Result::Err`), it might make sense if one squints.

Reply:
Good catch. I've updated it from "an error" to "None".